Tests/Fixtures/TestProject/xcode12_workaround.xcconfig

```diff
@@ -0,0 +1,8 @@
+//
+// See https://github.com/Carthage/Carthage/issues/3019
+//
+// Skips building ARM slices for simulators until Carthage can support it
+//
+
+EXCLUDED_ARCHS__EFFECTIVE_PLATFORM_SUFFIX_simulator__NATIVE_ARCH_64_BIT_x86_64=arm64 arm64e armv7 armv7s armv6 armv8
+EXCLUDED_ARCHS=$(inherited) $(EXCLUDED_ARCHS__EFFECTIVE_PLATFORM_SUFFIX_$(EFFECTIVE_PLATFORM_SUFFIX)__NATIVE_ARCH_64_BIT_$(NATIVE_ARCH_64_BIT))
```

Comment:
I don't think this will have any effect on Xcode versions before 12 (we use this same workaround internally)